src/session/sidebar/chat_row.rs

```diff
@@ -230,6 +230,14 @@ fn stringify_message(message: Message) -> String {
 
     let text_content = match message.content().0 {
         MessageContent::MessageText(data) => dim_and_escape(&data.text.text),
+        MessageContent::MessageBasicGroupChatCreate(data) => {
+            show_sender = false;
+            gettext!(
+                "{} created the group «{}»",
```

Comment:
```suggestion
                "{} created the group",
```

No need to specify the name here. This is also what Telegram for Android does (and it's also better for us having a shorter text because the sidebar can't be resized).